gulpfile.ts

```diff
@@ -1,130 +1,11 @@
 import gulp from 'gulp';
```

Comment:
Yes, it can also be generated automatically.

However, in the case of `package.json`, it seems better to use a "yarn workspace". I don't know exactly about this yet.

Reply:
No need, this library will not have external dependencies now and in the future, and the type of repository is not monorepo.

We just need to add name, version, description, license and entry to package json.

Reply:
Then I'll implement automatic generation of these. I have a schedule, so I think it will be difficult within today.

Reply:
Not today, this is not an urgent need.

You can it in your any free time. (no deadline)

Reply:
Thank you for your support, this library is just for personal hobby, you don't need to pay too much attention to it.

Just for fun. :)